### DISCUSSION TITLE
Presets question in Save Video node

### DISCUSSION THREAD

**Discussion by charliemagee** [chaiNNer-org/chaiNNer#2727] [Q&A]
Hello,

Anyone know what the presets for Save Video mean: Ultra Fast through Very Slow? Do those settings affect quality of output?

Also, typically things are Best at high numbers, like 5 Stars vs 1 Star. But Quality on this has Best = 1 and Worst = some higher number. Any particular reason?

**Comment by joeyballentine** [comment]
Since we use FFMPEG internally, those settings match the FFMPEG encoding and CRF settings. Here's some documentation for it: https://trac.ffmpeg.org/wiki/Encode/H.264
